packages/sync/Codec.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
+namespace Automattic\Jetpack\Sync;
+
 /**
  * Very simple interface for encoding and decoding input
  * This is used to provide compression and serialization to messages
  **/
-interface iJetpack_Sync_Codec {
+interface Codec {
```

Comment:
I think we should suffix interfaces with Interface. So that it is more clear that we are implementing one.

Reply:
I like that, I'll keep it in mind for the next iteration 👍